frontend/src/router.ts

```diff
@@ -163,23 +163,10 @@ export const hasClaim = (
   );
 };
 
-export const defaultFilterCb = (
-  route: RouteConfig,
-  userClaims: any,
-  requireClaims: any
-): any => {
-  if (route.meta.activator) {
-    const childRoutes = filterRoutesByClaims(defaultFilterCb)(
-      route.children,
-      userClaims,
-      requireClaims
-    );
-    if (childRoutes.length) return { ...route, children: childRoutes };
-  } else if (requireClaims) {
-    if (hasClaim(requireClaims, userClaims)) return route;
-  } else {
-    return route;
-  }
+export const getRouteChildren = () => {
```

Comment:
The purpose of this function is not clear. The children of which route does it return? I cannot understand this from the code.

Moreover, I doubt it should be exported. It is used only twice and in both cases it is used to be passed back to the other router function. Then why not to call it directly from the `getHomePageRoutes` and `getMenuItems` functions?